### PR TITLE
Add abort

### PR DESCRIPTION
**Pull request by yutakaoka**
issue #114 
I added ABORTED state for move_base action.
Could you take a look at my code?

Thanks,

### CODE REVIEW

**Comment by yutakaoka** [comment]
It seems like Lint error.

**Comment by at-wat** [comment]
Thank you for your contribution.

This package uses ROS recommended C++ style and tested by cpplint in roslint package.
http://wiki.ros.org/CppStyleGuide

`$ rosrun roslint cpplint path/to/source.cpp` reproduces the test result.

**Comment by yutakaoka** [comment]
Done.

File: planner_cspace/src/planner_3d.cpp [hunk 1]
```diff
@@ -1249,6 +1258,17 @@ class Planner3d
...
+          {
+            status_.error = planner_cspace::PlannerStatus::PATH_NOT_FOUND;
+            status_.status = planner_cspace::PlannerStatus::DONE;
+            has_goal_ = false;
+            act_->setAborted(move_base_msgs::MoveBaseResult(),
+                             "Goal is in Rock");
+            ROS_ERROR("Exceeded max_retry_num:%d", max_retry_num_);
+            continue;
+          }
+
```
Comment (at-wat) [inline planner_cspace/src/planner_3d.cpp]: It seems better to remove this empty line and add `{}` to `if (escaping_)` and `else` blocks.

File: planner_cspace/test/src/test_abort.cpp [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    if (!move_base_->waitForServer(ros::Duration(30.0)))
+    {
+      ROS_ERROR("Failed to connect move_base action");
+      exit(EXIT_FAILURE);
+    }
+  }
+  ~AbortTest() {}
+
+ protected:
+  typedef actionlib::SimpleActionClient<move_base_msgs::MoveBaseAction> ActionClient;
```
[the remark this answers is not in the document]
Comment (yutakaoka) [reply planner_cspace/test/src/test_abort.cpp]: Done

File: planner_cspace/src/planner_3d.cpp (hunk 1 above)
Comment (yutakaoka) [reply planner_cspace/src/planner_3d.cpp]: Done

**Pull request merged**